cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/CloudFormationTemplateBuilder.java

```diff
@@ -67,7 +67,6 @@ public String build(ModelContext context) {
                     group.getSecurity().isUseNetworkCidrAsSourceForDefaultRules(),
                     getInstanceProfile(group),
                     awsInstanceView.getOnDemandPercentage(),
-                    awsInstanceView.isFastEbsEncryptionEnabled(),
```

Comment:
I think we can not really remove this because of backward compatibility

Reply:
Resolving this based on the above conversation for `AwsGroupView.java` cc: @lajosrodek